resources/test/tests/functional/idlharness/IdlInterface/test_immutable_prototype.html

```diff
@@ -13,7 +13,16 @@
 </head>
 <body>
 <script>
-"use strictt";
+"use strict";
+var boundCallHasOwnProperty = Function.prototype.call.bind(Object.prototype.hasOwnProperty);
+
+// Workaround for: https://bugzilla.mozilla.org/show_bug.cgi?id=1688198
```

Comment:
What tests fail without this workaround? Are lots of idlharness.js tests going to start failing in Firefox?

Reply:
Actually just a few new failures, so this seems OK then:
https://wpt.fyi/results/?diff&filter=ADC&run_id=5731820836487168&run_id=5665408696713216

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1688198 should be fixed pretty soon.